src/transaction_call_builder.ts

```diff
@@ -66,9 +64,17 @@ export class TransactionCallBuilder extends CallBuilder<
   public forLedger(sequence: number | string): this {
     const ledgerSequence =
       typeof sequence === "number" ? sequence.toString() : sequence;
+    return this.forEndpoint("ledgers", ledgerSequence);
```

Comment:
Nit: maybe what you did in other method:
```suggestion
    return this.forEndpoint("ledgers", sequence.toString());
```

Reply:
Will be fixed by #702 :+1: